docker-compose.prod.yml

```
@@ -0,0 +1,48 @@
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 40s
+
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "3"
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (2 > 0)

(empty-lines)